app/controllers/sessions_controller.rb

```diff
@@ -5,13 +5,15 @@ def new
   def create
     @user = User.find_by(email: params[:session][:email].downcase)
     if @user&.authenticate(params[:session][:password])
-      logged_in @user
-      params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
-      redirect_back_or @user
-    else
-      flash.now[:danger] = t "invalid_email_password_combination"
-      render :new
-    end
+        if @user.activated
+          log_in @user
+          params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
+          redirect_back_or @user
+        else
+          message = t(:error_acc_unacti)
+          flash[:warning] = t(:message)
+          redirect_to root_url
+        end
```

Comment:
![2021-09-07_10-03](https://user-images.githubusercontent.com/72131384/132277803-29651863-b02c-49dd-9cb4-df2bde3818e9.png)
Action create should be like this pic